app/controllers/animal_sightings_controller.rb

```diff
@@ -0,0 +1,53 @@
+class AnimalSightingsController < ApplicationController
+      def index
+            sightings = AnimalSightings.all
+            render json: sightings
+      end
+
+      def show
+            sighting = AnimalSighting.find(params[:id])
+            render json: sighting
+      end
+
+      # def new
+
+      # end
```

Comment:
Since we won't need the new and edit methods for an API, please remove the commented-out code.